content/en-us/reference/engine/classes/CaptureService.yaml

```diff
@@ -51,14 +51,14 @@ methods:
     security: None
     thread_safety: Unsafe
   - name: CaptureService:PromptShareCapture
-    summary: ''
-    description: ''
+    summary: "Ask to share screenshots with the player's contacts."
```

Comment:
It may be better to specify that this prompts the user to share via a share sheet, contacts is ambiguous.